advanced/Scripts/query.sh

```diff
@@ -133,22 +116,17 @@ fi
 
 # Scan Wildcards
 if [[ -e "${wildcardlist}" ]]; then
-    # Determine all subdomains, domain and TLDs
-    mapfile -t wildcards <<< "$(processWildcards "${domainQuery}")"
-    for match in "${wildcards[@]}"; do
-        # Search wildcard list for matches
-        mapfile -t results <<< "$(scanList "${match}" "${wildcardlist}" "wc")"
-        if [[ -n "${results[*]}" ]]; then
-            if [[ -z "${wcMatch:-}" ]] && [[ -z "${blockpage}" ]]; then
-                wcMatch=true
-                echo " ${matchType^} found in ${COL_BOLD}Wildcards${COL_NC}:"
-            fi
-            case "${blockpage}" in
-                true ) echo "π ${wildcardlist##*/}"; exit 0;;
-                *    ) echo "   *.${match}";;
-            esac
+    mapfile -t results <<< "$(scanList "${domainQuery}" "${wildcardlist}" "wc")"
+    if [[ -n "${results[*]}" ]]; then
+        if [[ -z "${wcMatch:-}" ]] && [[ -z "${blockpage}" ]]; then
+            wcMatch=true
+            echo " ${matchType^} found in ${COL_BOLD}Wildcards${COL_NC}:"
```

Comment:
This should say "Regex" instead of "Wildcards" since using "wildcards" does not fit with the rest of the software.